Improve doc traps snmp (#7048)
centreon/centreon: doc/en/configuration_guide/advanced_configuration/traps.rst

.. note:: the Centcore process is responsible, as for the export of configuration of the supervision, to copy the SQLite base on the remote collector.

> Order of achievement of actions by the centreontrapd process

Reviewer: ```suggestion
Successive actions by the centreontrapd process
```

============================================================

> The order of achievement of actions by the centreontrapd process is:

Reviewer: ```suggestion
Successive actions by the centreontrapd process are:
```

.. image :: /images/user/configuration/10advanced_configuration/SNMP_Traps_management_general_view.png
    :align: center

*************************

> Configuration of services

Reviewer: ```suggestion
Configuring services
```

> Configuration of the connection to the database

Reviewer: ```suggestion
Configuring the database connection
```

> * If the **Reschedule associated services** box is checked the next check on the service, which should be ‘active’, should be reprogrammed as soon as possible after reception of the trap.

Reviewer: ```suggestion
* If the **Reschedule associated services** box is checked, the next check on the service, which should be ‘active’, should be reprogrammed as soon as possible after receiving the trap.
```

> * If the **Execute special command** box is checked the command defined in Special command is executed.

Reviewer: ```suggestion
* If the **Execute special command** box is checked, the command defined in Special command is executed.
```

> Very advanced configuration of the traps - Routing

Reviewer: ```suggestion
Advanced trap configuration - Routing
```

*       The **Execution Type** field is used to enable the Execution interval by defining the conditions by Root OID, by the Root OID and host combination or, to disable this restriction, None.
*       The **Execution Method** field is used to define if on reception of multiple same events (Root OID). The execution is either **Sequential** or **Parallel**.

> Very advanced configuration of the traps - Custom code

Reviewer: ```suggestion
Advanced trap configuration - Custom code
```

------------------------------------------------------

> The field **custom code** allows to run custom Perl processing.

Reviewer: ```suggestion
The field **custom code** allows custom Perl processing.
```

To enable this feature, you must set **secure_mode** to 0 in
**/etc/centreon/centreontrapd.pm** file as: ::

    our %centreontrapd_config = (
       ...
       secure_mode => 0,
       ....
    );
    
    1;

> For example, to decode the 4 argument from hexadecimal, the custom code will be: ::

Reviewer: ```suggestion
For example, to decode the 4 arguments from hexadecimal, the custom code will be: ::
```
